torchtune/_recipe_registry.py

```diff
@@ -28,16 +28,12 @@ class Recipe:
         file_path="full_finetune_single_device.py",
         configs=[
             Config(
-                name="llama2/7B_full_single_device",
-                file_path="llama2/7B_full_single_device.yaml",
+                name="llama2/7B_full",
+                file_path="llama2/7B_full.yaml",
             ),
             Config(
-                name="llama2/7B_full_single_device_low_memory",
-                file_path="llama2/7B_full_single_device_low_memory.yaml",
-            ),
-            Config(
-                name="mistral/7B_full",
-                file_path="mistral/7B_full.yaml",
+                name="llama2/7B_full_low_memory",
+                file_path="llama2/7B_full_low_memory.yaml",
```

Comment:
So mistral recipes are distributed only now?

Reply:
You can run them --nproc_per_node as 1 :)

Reply:
I guess, but is this really the intent? Why not just keep them supported for single device recipe if it's really all the same?

Reply:
Added back support